src/momento/simple_cache_client.py

```diff
@@ -135,12 +148,6 @@ def create_cache(self, cache_name: str) -> CreateCacheResponse:
 
         Returns:
             CreateCacheResponse:
-
-            This response is resolved to one of:
-
-            - `CreateCache.Success`
-            - `CreateCache.AlreadyExists`
-            - `CreateCache.Error`
```

Comment:
Are these changes in the async code?

Reply:
No. When I deleted them earlier I forgot to make gen-sync before committing. I don't think we'll conflict, but I can put up a separate PR to fix it.